engine/requirements-dev.txt

```diff
@@ -25,14 +25,14 @@ django==4.2.16
     #   django-stubs-ext
 django-filter-stubs==0.1.3
     # via -r requirements-dev.in
-django-stubs[compatible-mypy]==4.2.2
```

Comment:
I ran `uv pip compile requirements.in -o requirements.txt` and `uv pip compile requirements-dev.in -o requirements-dev.txt` and it unexpectedly changed a bunch of lines. These `package[extra]` entries seem to be all commited by dependabot at some point. I suspect this is a dependabot issue, it should not add the `[extra]` part into the "compiled" dependency list. Anyway, I think these changes don't actually change any dependencies.

Reply:
was going to ask about this, I would say as long as `mypy` is passing on CI, should be okay 👍